libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/ProfileShort.kt

```diff
@@ -0,0 +1,55 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Profile short message options
+ *
+ * @param options common options for profile short message
+ * @param types options for each type of profile short message
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class ProfileShort(
```

Comment:
internal constructor + builder
Otherwise it'll be more difficult to extend the API.
But probably also not critical for the snapshot.